docs/vectorizer-quick-start.md

```diff
@@ -1,52 +1,60 @@
 # Vectorizer quick start
 
-This page shows you how to create a vectorizer in a self-hosted Postgres instance, then use 
-the pgai vectorizer worker to create embeddings from data in your database. To finish off we show how simple it 
-is to do a semantic search query on the embedded data in one query!
+This page shows you how to create an ollama-based vectorizer in a self-hosted Postgres instance. We also show how simple it is to do semantic search on the automatically embedded data!  
+If you prefer working with the openai-api instead of self hosting models, you can jump over to the [openai quick start](vectorizer-quick-start-openai.md).
 
-## Setup a local developer environment
+## Setup a local development environment
 
-The local developer environment is a docker configuration you use to develop and test pgai, vectorizers and vectorizer
-worker locally. It includes a:
+We use a docker-compose file to set up a development environment, it includes a:
 - Postgres deployment image with the TimescaleDB and pgai extensions installed
 - pgai vectorizer worker image
+- ollama image to host embedding and large language models
 
 On your local machine:
 
 1. **Create the Docker configuration for a local developer environment**
 
-   Add the following docker configuration to `<timescale-folder>/docker-compose.yml`:
+   Create the following `docker-compose.yml` in a new directory:
     ```yaml
-    name: pgai
-    services:
-      db:
-        image: timescale/timescaledb-ha:pg16
-        environment:
-          POSTGRES_PASSWORD: postgres
-          OPENAI_API_KEY: <your-api-key>
-        ports:
-          - "5432:5432"
-        volumes:
-          - ./data:/var/lib/postgresql/data
-      vectorizer-worker:
-        image: timescale/pgai-vectorizer-worker:0.1.0
-        environment:
-          PGAI_VECTORIZER_WORKER_DB_URL: postgres://postgres:postgres@db:5432/postgres
-          OPENAI_API_KEY: <your-api-key>
+   name: pgai 
+   services:
+     db:
+       build:
+         dockerfile: Dockerfile
+         context: ../pgai/projects/extension
+         target: pgai-test-db
+       environment:
+         POSTGRES_PASSWORD: postgres
+       ports:
+         - "5432:5432"
+       volumes:
+         - ./data:/var/lib/postgresql/data
+     vectorizer-worker:
+       build:
+         dockerfile: Dockerfile
+         context: ../pgai/projects/pgai
+       environment:
+         PGAI_VECTORIZER_WORKER_DB_URL: postgres://postgres:postgres@db:5432/postgres
+         OLLAMA_HOST: http://ollama:11434
+       command: [ "--poll-interval", "5s" ]
+     ollama:
+       image: ollama/ollama
     ```
 
-1. **Tune the developer image for your AI provider**
-
-   Replace the instances of `OPENAI_API_KEY` with a key from your AI provider.
-
-1. **Start the database**
+1. **Start the services**
    ```shell
-    docker compose up -d db
+    docker compose up -d
```

Comment:
Note: this still won't work, because the vectorizer worker isn't robust to failures in connecting to the DB etc. I've opened a PR which fixes this: https://github.com/timescale/pgai/pull/263

Reply:
done